hardware.inc

```diff
@@ -25,40 +25,45 @@
 ;* Rev 2.8 - 03-Feb-19 : Added audio registers flags (Álvaro Cuesta)
 ;* Rev 2.9 - 28-Feb-20 : Added utility rP1 constants
 ;* Rev 3.0 - 27-Aug-20 : Register ordering, byte-based sizes, OAM additions, general cleanup (Blitter Object)
+;* Rev 4.0 - 03-May-21 : Updated to use RGBDS 0.5.0 syntax, changed IEF_LCDC to IEF_STAT (Eievui)
 
 ; If all of these are already defined, don't do it again.
 
     IF !DEF(HARDWARE_INC)
-HARDWARE_INC SET 1
+DEF HARDWARE_INC SET 1
 
-rev_Check_hardware_inc : MACRO
+IF __RGBDS_MAJOR__ == 0 && __RGBDS_MINOR__ < 5

```

Comment:
This check needs to be above the `DEF` line if we want to avoid a fatal syntax error.